samples/notebooks/prediction/SDK_Custom_Predict_and_Handler_SDK_Integration.ipynb

```diff
@@ -0,0 +1,1434 @@
+    "\n",
+    "    def predict(self, instances):\n",
+    "        \"\"\"Performs prediction.\"\"\"\n",
+    "        instances = instances[\"instances\"]\n",
+    "        inputs = np.asarray(instances)\n",
+    "        preprocessed_inputs = self._preprocessor.preprocess(inputs)\n",
+    "        outputs = self._model.predict(preprocessed_inputs)\n",
+    "\n",
+    "        return {\"predictions\": [self._class_names[class_num] for class_num in outputs]}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then implement a custom `Handler`.\n",
+    "\n",
+    "A [`Handler`](https://github.com/googleapis/python-aiplatform/blob/custom-prediction-routine/google/cloud/aiplatform/prediction/handler.py) must implement the following interface."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class Handler:\n",
+    "    \"\"\"Interface for Handler class to handle prediction requests.\"\"\"\n",
+    "\n",
+    "    def __init__(\n",
+    "        self, gcs_artifacts_uri: str, predictor: Optional[Type[Predictor]] = None,\n",
+    "    ):\n",
+    "        \"\"\"Initializes a Handler instance.\n",
+    "        Args:\n",
+    "            gcs_artifacts_uri (str):\n",
+    "                Required. The value of the environment variable AIP_STORAGE_URI.\n",
+    "            predictor (Type[Predictor]):\n",
+    "                Optional. The Predictor class this handler uses to initiate predictor\n",
+    "                instance if given.\n",
+    "        \"\"\"\n",
+    "        pass\n",
+    "\n",
+    "    def handle(self, request: Request) -> Response:\n",
+    "        \"\"\"Handles a prediction request.\n",
+    "        Args:\n",
+    "            request (Request):\n",
+    "                The request sent to the application.\n",
+    "        Returns:\n",
+    "            The response of the prediction request.\n",
+    "        \"\"\"\n",
+    "        pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then implement a custom `Handler`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile $USER_SRC_DIR/handler.py\n",
+    "\n",
+    "import csv\n",
+    "from io import StringIO\n",
+    "import json\n",
+    "\n",
+    "from fastapi import Response\n",
+    "\n",
+    "from google.cloud.aiplatform.prediction.handler import Handler\n",
+    "\n",
+    "from typing import Optional, Type\n",
+    "\n",
+    "class CprHandler(Handler):\n",
+    "    \"\"\"Default prediction handler for the prediction requests sent to the application.\"\"\"\n",
+    "\n",
+    "    def __init__(\n",
+    "        self, gcs_artifacts_uri, predictor = None,\n",
+    "    ):\n",
+    "        \"\"\"Initializes a Handler instance.\"\"\"\n",
+    "        if predictor is None:\n",
```

Comment:
Can we inherit from PredictionHandler, and not implement this part?

Reply:
Done.